gcp/src/main/java/org/apache/iceberg/gcp/GCPProperties.java

```diff
@@ -43,6 +43,8 @@ public class GCPProperties implements Serializable {
   public static final String GCS_OAUTH2_TOKEN_EXPIRES_AT = "gcs.oauth2.token-expires-at";
   // Boolean to explicitly configure "no authentication" for testing purposes using a GCS emulator
   public static final String GCS_NO_AUTH = "gcs.no-auth";
+  public static final String GCS_OAUTH2_REFRESH_CREDENTIALS_ENDPOINT =
```

Comment:
we most likely also need a property to explicitly disable refreshing vended creds for Kafka Connect